docs/configuration/config-file-reference.md

```diff
@@ -183,6 +183,40 @@ The `server_config` configures the HTTP and gRPC server of the launched service(
 # CLI flag: -server.grpc-conn-limit
 [grpc_listen_conn_limit: <int> | default = 0]
 
+http_tls_config:
```

Comment:
Is this file supposed to be updated as part of this PR

Reply:
Yes, because of the CI linter.

Reply:
Ahh, this was added by the weaveworks common server. Not a remnant of the other https client PR.